feat(homepage): replace Twitter button with links to OAI-PMH and Z39.…

camillernd opened the pull request:
# feat: replace Twitter icon with OAI-PMH and Z39.50 server links

## 🤔 What

* Removed the Twitter icon from the homepage footer
* Added two new buttons linking to:

  * the **OAI-PMH server** (`REACT_APP_OAI_URL`)
  * the **Z39.50 server** (`REACT_APP_Z3950_URL`)
* Created and added two new icons in `/public/images/icons8/` following the existing design
* Updated the `.env` file with the two new variables

## 🤷‍♂️ Why

This update was requested to give visibility to the OAI-PMH and Z39.50 services.

## 🔍 How

* Updated the SocialLinks.tsx component by commenting out the Twitter block and adding two new list items for the OAI-PMH and Z39.50 links.
* Followed the structure and styling of the other social buttons.
* Icons follow the same styling convention and are hosted locally.
* Links are driven by environment variables so they can be easily updated post-deployment.

## 🧪 Testing

* Run `yarn start`
* Scroll down to the homepage footer
* Check:

  * Twitter icon is gone
  * OAI and Z39.50 buttons appear
  * Clicks open in new tab (currently `"#"` until real URLs are set)
* You can test with custom URLs by editing your `.env` file:

```env
REACT_APP_OAI_URL=https://example.com/oai
REACT_APP_Z3950_URL=https://example.com/z3950
```

## 📸 Preview
![image](https://github.com/user-attachments/assets/ec25cccb-352b-4051-a9bb-d4db923c464d)

---

camillernd committed:
feat(homepage): replace Twitter button with links to OAI-PMH and Z39.50 servers

urien commented:
Parfait Camille. Je te propose de mettre directement les liens vers
https://oai.grottocenter.org
https://z39-50.grottocenter.org

Tu publieras ce travail quand le serveur sera en production
Eventuellement est ce que tu peux faire une infobulle comme pour les licences afin de préciser ce que sont ces services quand on passe la souris sur le picto. Il faut utiliser les outils de localisation pour qu'on ai une traduction dans la langue utilisée pour le site

camillernd committed:
feat(homepage): add localized tooltip for OAI-PHM and Z39.50 server links and make links point to production servers

camillernd commented:
## 🤔 What

Added tooltips to the OAI-PMH and Z39.50 buttons on the homepage footer using the localization system.  
Set the real production URLs for both services:

- https://oai.grottocenter.org  
- https://z39-50.grottocenter.org

---

## 🧩 Why

Following the request from Frédéric, the buttons now include a tooltip (like the API license icon) and point to the production servers.

---

## 🔍 How

- Defined new localized strings in `lang/en.json`, `lang/fr.json`, etc.:
  - `oaiTooltip`
  - `z3950Tooltip`

- Updated the `SocialLinks.jsx` component to:
  - Use `formatMessage` for localized tooltips

- Updated the `.env` to:
  - Replace temporary `"#"` links with production URLs

---

## 🧪 Testing

1. Run `yarn start`
2. Scroll down to the footer
3. Hover over the OAI and Z39.50 icons:
   - ✅ Tooltips should appear in your selected language
   - ✅ Click opens the correct production link in a new tab

---

## 📸 Preview
![image](https://github.com/user-attachments/assets/4e75733c-02db-4c8c-912d-3aa2918c00bc)
![image](https://github.com/user-attachments/assets/eeea7018-de84-438c-bcb6-9d66e5000f5c)

urien reviewed (approved):
Excuses moi Camille mais on n'arrive pas sur la page d'accueil html des serveurs. est ce que tu peux changer les liens avant de publier ?
Merci d'avance

camillernd commented:
> Excuses moi Camille mais on n'arrive pas sur la page d'accueil html des serveurs. est ce que tu peux changer les liens avant de publier ? Merci d'avance

Pour l'instant la page d'accueil OAI est à https://oai.grottocenter.org/scicat/home/
je vais modifier ça pour qu'elle soit bien disponible à https://oai.grottocenter.org/

Idem pour le Z3950 c'est à https://oai.grottocenter.org/scicat/home/Z3950.html
Il faudrait que ce soit idéalement https://z3950.grottocenter.org/

Je vais faire les modifs nécéssaires sur les serveurs avant de merge.
